Apps/phciscothreatresponse/ciscothreatresponse_connector.py

```diff
@@ -0,0 +1,443 @@
+# -----------------------------------------
```

Comment:
Missing copyright header

Reply:
@alexa-phantom It is a community app. So we do not add the our copyrights

Reply:
I can see copyrights of Splunk in other files. I will add them in missing files

Reply:
@alexa-phantom Fixed.